xml/System.IO/RandomAccess.xml

```diff
@@ -14,8 +14,12 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Provides offset-based APIs for reading and writing files, ensuring atomic and thread-safe operations.</summary>
```

Comment:
@adamsitnik This class didn't have a description, so I took some inspiration from the first API proposal you wrote:

https://github.com/dotnet/runtime/issues/24847#issuecomment-807050146

And also from the PR description:

https://github.com/dotnet/runtime/pull/53669

Do you agree with the provided information?

Reply:
I was initailly using the "atomic" word in the issue description, as I meant lack of two separate operations: seek to given offset and read|write from current offset. But after I've learned about the existence of [CreateFileTransacted](https://docs.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-createfiletransacteda) and some of the file system complexities (the fact that write has succeeded does not mean that data is going to be stored on the disk) I believe that it's better to not use this word. Thread-safe is all we need.

Sorry for not adding that in the first place when I was adding `RandomAccess` type.

```suggestion
    <summary>Provides offset-based APIs for reading and writing files in a thread-safe manner.</summary>
```